packages/admin/admin-theme/src/componentsTheme/MuiDataGrid.tsx

```diff
@@ -113,12 +131,34 @@ export const getMuiDataGrid: GetMuiComponentTheme<"MuiDataGrid"> = (component, {
                 width: 16,
                 height: 16,
             },
+
+            ["@media (max-width: 900px)"]: {
```

Comment:
@jamesricky shouldn't it be possible to use `theme.md.down()` or similar here?

Reply:
It should be, but it's not 😅 

We'll need to add a separate `breakpoints.ts` file like we have for the shadows, then we can pass the breakpoint values to the `themeData` used here. I'll create a task for that but I think we can leave it as is here for now.

Reply:
FYI: COM-568